NEWS

```diff
@@ -1,3 +1,15 @@
+26
+
+Update sugargame
+Fix functions taking tuple parameters
```

Comment:
Can you add the `*` before each item on the changelog, similar to what was done in v25?

Reply:
Oops missed that, fixed